release.properties

```diff
@@ -0,0 +1,11 @@
+#release configuration
```

Comment:
hmm where did this come from? I'd probably go without this to keep things consistent